tests/helpers.mjs

```diff
@@ -16,18 +16,30 @@ const initialOperatorStake = 100_000_000_000;
 export const basePermabuyPrice = 2_500_000_000;
 export const baseLeasePrice = 600_000_000;
 
+export const genesisEpochTimestamp = 1719900000000; // Tuesday, July 2, 2024, 06:00:00 AM UTC
+export const epochLength = 1000 * 60 * 60 * 24; // 24 hours
+export const distributionDelay = 1000 * 60 * 40; // 40 minutes
+
 const { handle: originalHandle, memory } = await createAosLoader();
 export const startMemory = memory;
 
-export async function handle(options = {}, mem = startMemory) {
-  return originalHandle(
+export async function handle(
```

Comment:
Was it going to be too big a lift to move to an object for args? All the `, undefined, false)` that's starting to appear in the code is a bit unsightly...

Reply:
yeah it seemed large, but should be doable -- I'll timebox it

Reply:
[0b61e65](https://github.com/ar-io/ar-io-network-process/pull/250/commits/0b61e65340fcd055492b9d96d2a7bbd781198be0)